.github/workflows/build-test-image.yaml

```
@@ -9,19 +9,19 @@ on:
   workflow_dispatch:
 
 concurrency:
-  group: falkordb-node-branch-${{ github.event_name == 'pull_request' && github.event.pull_request.head.label || github.ref_name }}
+  group: falkordb-node-branch-${{ github.event_name == 'pull_request' && github.ref_name.event.pull_request.head.label || github.ref_name }}
```

actionlint

receiver of object dereference "event" must be type of object but got "string"

(expression)